opencti-platform/opencti-graphql/src/domain/enrichment.js

```diff
@@ -40,3 +46,35 @@ export const createEntityAutoEnrichment = async (context, user, element, scope)
   );
   return workList;
 };
+
+const findConnectorsForElementEnrichment = async (context, user, element, scope) => {
+  const connectors = await getEntitiesListFromCache(context, user, ENTITY_TYPE_CONNECTOR);
+  return filterConnectorsForElementEnrichment(context, connectors, element, scope);
+};
+
+export const filterConnectorsForElementEnrichment = async (context, connectors, element, scope) => {
+  // first filter active & enrichment connectors only
+  const activeConnectors = connectors.filter((conn) => conn.active === true && conn.connector_type === CONNECTOR_INTERNAL_ENRICHMENT);
+  const targetConnectors = [];
+  for (let i = 0; i < activeConnectors.length; i += 1) {
+    const conn = activeConnectors[i];
+    const scopeMatch = scope ? (conn.connector_scope ?? []).some((s) => s.toLowerCase() === scope.toLowerCase()) : true;
+    const autoTrigger = conn.auto === true || (conn.connector_trigger_filters && await isStixMatchConnectorFilter(context, element, conn.connector_trigger_filters));
```

Comment:
shouldn't it be something like 
`conn.connector_trigger_filters ?? await isStixMatchConnectorFilter(context, element, conn.connector_trigger_filters) : conn.auto === true` ?
if we want to override connector auto config?

Reply:
yes you're right